src/adagrams.js

```diff
@@ -1,15 +1,138 @@
+const letterPool = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 2,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+const scoringSystem = {
+  A: 1,
+  B: 3,
+  C: 3,
+  D: 2,
+  E: 1,
+  F: 4,
+  G: 2,
+  H: 4,
+  I: 1,
+  J: 8,
+  K: 5,
+  L: 1,
+  M: 3,
+  N: 1,
+  O: 1,
+  P: 3,
+  Q: 10,
+  R: 1,
+  S: 1,
+  T: 1,
+  U: 1,
+  V: 4,
+  W: 4,
+  X: 8,
+  Y: 4,
+  Z: 10,
+};
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const generatedPool = generatePool();
+  const hand = [];
+
+  function generatePool() {
+    const newArr = [];
+    Object.entries(letterPool).forEach(([key, value]) => {
+      newArr.push(repeatElement(key, value));
+    });
+    return newArr.flat();
+  }
+
+  function repeatElement(element, count) {
+    return Array(count).fill(element).flat();
+  }
+
+  function drawLetter() {
```

Comment:
Hi Elaine! Great job completing JS Adagrams! Congrats on finishing your first JavaScript project 🎉 Excellent choice with this data structure & weighting approach